### ISSUE TITLE
Model Convergence Verification Scripts

### ISSUE THREAD

**Issue by yaoyu-33** [NVIDIA-NeMo/Megatron-Bridge#870]
We want to create some auto sft/lora scripts on Squad to cross check with Automodel for loss start and loss end values. Also in bridge, we need to additionally check model parallel combinations, maybe create a table automatically in the end and people can attach in the model support issue/ PR.

https://docs.google.com/spreadsheets/d/17YwnO3_XloYaNKW4IS9VbEE_SS9gyXc2kJK3IjL37_E/edit?gid=1959706898#gid=1959706898 (NV Internal Link)

**Comment by yaoyu-33** [comment]
Potential datasets:
LLM pretrain: https://github.com/mlcommons/training/tree/master/large_language_model_pretraining/nemo
https://gitlab-master.nvidia.com/dl/mlperf/optimized/-/blob/main/llama31_405b/pytorch/config_data_eos.sh?ref_type=heads (internal)

VLM finetune: cord-v2
